src/framework/mod.rs

```diff
@@ -36,6 +36,7 @@ pub enum SearchMatch {
 #[derive(Debug)]
 pub enum Environment {
     Production,
+    Dev(url::Url),
```

Comment:
Not sure if `Dev` is better or `Custom` is better.

Reply:
I thought Dev was better, but now I actually prefer Custom, for no real reason.